src/models/character.ts

```diff
@@ -29,4 +36,38 @@ export class Character {
             
         return valid;
     }
+    
+    fromDatabase(data: any[]): Character {
+        const db_data: any = data[0];
+        this.id = db_data.id;
+        this.owner_id = db_data.owner_id;
+        this.class_id = db_data.class_id;
+        this.race_id = db_data.race_id;
+        this.hitpoints = db_data.hitpoints;
+        this.raw_stats = db_data.raw_stats;
+        this.mod_stats = db_data.mod_stats;
+        this.background = db_data.background;
+        this.alignment = db_data.alignment;
+        this.description = db_data.description;
+        this.inventory = db_data.inventory;
+
+        return this;
+    }
+
+    insertParams(): any[] {
+        return [
+            this.owner_id, this.name, this.class_id, this.race_id, this.hitpoints, this.raw_stats, 
+            this.mod_stats, this.background, this.alignment, this.description
+        ];
+    }
+
+    insertString(): string {
```

Comment:
`insertString` doesn't seem to match the order of `insertParams`. `name` is only in the params and `inventory` is only in the insertString.

Reply:
Thanks, I fixed all of the `params` and `string` fields in this class, as well as the fromDatabase. For some reason all of them were missing name, and the order was weird. I made all of them the same order as the attributes at the top of the file.

I'm sure there's an easier way to do this, but I'm not sure what it is